MovieApp/PresentationLayer/Favorites/FavoritesViewController.swift

```diff
@@ -14,6 +15,8 @@ class FavoritesViewController: UIViewController {
 
     private var movies: [DetailedMovieViewModel] = []
 
+    private var cancellable: AnyCancellable?
```

Comment:
Use `private var disposables = Set<AnyCancellable>()` instead. And below, right after `sink`, call `.store(in: &disposables)`